Problem3.java

```diff
@@ -0,0 +1,21 @@
+//Time Complexity - O(n)
+//Space Complexity - O(n)
+// This solution worked on LeetCode
```

Comment:
Space complexity will be constant as you are storing characters in a hashset, there will be 26 characters at most.